server/routes/inPrisonCoursesAndQualifications/index.ts

```diff
@@ -31,12 +30,6 @@ export default (router: Router, services: Services) => {
 
   // Route for use when being linked to from DPS Prisoner Profile within the context of a prisoner's Work and Skills section of their Prisoner Profile
   router.get('/prisoner/:prisonNumber/work-and-skills/in-prison-courses-and-qualifications', [
-    checkPrisonerInCaseload({
```

Comment:
We've had a suitable middleware that checks a prisoner is in your caseload for some time. Previously it was only used on this route, which is a "special" route that is linked to directly from the DPS Prisoner Profile page.

(The middleware was introduced a while back, and functionally does exactly what a similar middleware in DPS Prisoner Profile does (I copied it from there 😁  👍 ))